services/QuillLMS/app/models/classlink_classroom_user.rb

```diff
@@ -20,6 +20,9 @@
 # This class should be called ClasslinkClassroomsStudent instead of ClasslinkClassroomUser.
 # But sticking to existing convention.
 class ClasslinkClassroomUser < ProviderClassroomUser
+  validates :classroom_external_id, length: { maximum: 200 }
```

Comment:
200 is my semi-arbitrary cutoff. Anything longer than that is probably degenerate. If we go too short, we risk error'ing on valid data.